demo/demo_FM_spinwaves.m

```diff
@@ -27,8 +27,9 @@
 js=pars(1); delta=pars(2);
 gam=pars(3); temp=pars(4); amp=pars(5);
 
-omega0 = delta + ...
-    (4*js)*((sin(pi.*qh)).^2 + (sin(pi.*qk)).^2 + (sin(pi.*ql)).^2);
+%omega0 = delta + ...
+%    (4*js)*((sin(pi.*qh)).^2 + (sin(pi.*qk)).^2 + (sin(pi.*ql)).^2);
+omega0 = delta + (8*js)*(1-cos(pi.*qh).*cos(pi.*qk).*cos(pi.*ql));
```

Comment:
you have left the old omega0 formulation commented out in the code. Unclear if your new formulation is just  a condensed rewrite of the old or a correction. Prefer you comment on what the change means